pytorch_to_returnn/torch/nn/modules/operator.py

```diff
@@ -394,6 +406,22 @@ def create_returnn_layer_dict(self, input: Tensor) -> Dict[str, Any]:
     return {"class": "reduce", "mode": self.mode, "axes": self.axes, "from": self._get_input_layer_name(input)}
 
 
+class Length(Module):
+  """
+  Wraps RETURNN LengthLayer.
+  """
+  is_original_torch_module = False
+
+  def __init__(self, axis: Union[str, Dim]):
+    super(Length, self).__init__()
+    self.axis = axis
+
+  def create_returnn_layer_dict(self, input: Tensor) -> Dict[str, Any]:
+    return {
+      "class": "length", "axis": self._get_input_axis_to_returnn(input, self.axis),
```

Comment:
When we also have `axis` as `Dim`, then:
```suggestion
      "class": "length", "axis": axis if isinstance(axis, Dim) else self._get_input_axis_to_returnn(input, self.axis),
```

Reply:
Note that `_get_input_axis_to_returnn` expects an `int` (PyTorch axis), nothing else, not a `str` and also not a `Dim`.

Reply:
`str` should also work, right? Maybe we would like to get batch shape so we have `axis="B"`?

Reply:
No, `_get_input_axis_to_returnn` always expects an `int`.
But anyway, see my follow-up comment. I think we always should make `axis` an `int`.

Reply:
We allow only `int` now, see comments below.